src/components/ui/SearchSuggestions.tsx

```
@@ -0,0 +1,41 @@
+
+export function SearchSuggestions({
+  suggestions,
+  onSuggestionSelect,
+  className,
+  ...props
+}: SearchSuggestionsProps) {
+  if (suggestions.length === 0) return null;
+
+  return (
+    <ul
+      role="listbox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)